ion/symboltoken.go

```diff
@@ -103,6 +113,15 @@ func symbolIdentifier(symbolText string) (int64, bool) {
 	return SymbolIDUnknown, false
 }
 
+// NewSimpleSymbolToken returns a Symbol Token with the given text value and undefined SID and Source.
+func NewSimpleSymbolToken(text string) SymbolToken {
+	return SymbolToken{Text: &text, LocalSID: SymbolIDUnknown}
+}
+
+func newSimpleSymbolTokenPtr(text string) *SymbolToken {
```

Comment:
Personally, I think we should not have this. we have ```NewSimpleSymbolToken()``` and if its pointer is needed that should be sufficient.

Reply:
I would love to not have this either but without it I would need two lines to get the address.

ie.
```go
st := NewSimpleSymbolToken(text)
&st
```
instead of
```go
newSimpleSymbolTokenPtr(text)
```